.circleci/config.yml

```diff
@@ -32,56 +43,44 @@ jobs:
             - _build
             - deps
 
-  build_elixir_1_13_otp_24:
-    docker:
-      - image: hexpm/elixir:1.13.1-erlang-24.2-alpine-3.15.0
-    steps:
-      - checkout
-      - <<: *install_hex_rebar
-      - restore_cache:
-          keys:
-            - v1-mix-cache-{{ checksum "mix.lock" }}
-      - run: mix deps.get
-      - run: mix compile
-
-  build_elixir_1_12_otp_24:
+  build_elixir_1_14_otp_25:
     docker:
-      - image: hexpm/elixir:1.12.1-erlang-24.0.2-alpine-3.13.3
+      - image: hexpm/elixir:1.14.3-erlang-25.2.3-alpine-3.18.0
+    <<: *defaults
     steps:
+      - <<: *install_system_deps
       - checkout
       - <<: *install_hex_rebar
-      - restore_cache:
-          keys:
-            - v1-mix-cache-{{ checksum "mix.lock" }}
       - run: mix deps.get
       - run: mix compile
+      - run: mix test
+      - run: mix dialyzer
```

Comment:
I don't think we nee dialyzer in 1.14. Instead, dialyzer needs to be updated to the latest to support OTP 26 and then this should be good to go

Reply:
A quick sanity check. Removed